src/components/DropableContainer/DropableContainer.styles.ts

```diff
@@ -42,6 +42,8 @@ export const Container = styled.div<{
   font-size: 14px;
   line-height: 17px;
   font-family: 'Lato Regular', sans-serif;
+  width: 150px;
+  justify-content: center;
```

Comment:
<DropableContainer> has been extracted to `keen.io/ui-core`, the width should be provided by props if required